### DISCUSSION TITLE
我想更换生成的 html 内容，请问应该怎么做？

### DISCUSSION THREAD

**Discussion by ifeisier** [hexojs/hexo#4882] [Question / Answer] [answered]
修改 markdown 到 html 的 document。

**Comment by libreliu** [comment, marked as answer]
如果在用 `hexo-renderer-marked`，可以考虑写插件继承 marker.Renderer 类并且 override 其中的方法。

**Comment by ifeisier** [comment]
谢谢

libreliu ***@***.***> 于2022年8月26日周五 00:34写道：
